docs/src/vertical/sbm.md

```diff
@@ -210,22 +313,71 @@ zone layer(s).
+determined from the minimum of `ksat`, actual transpiration `actevapustore` [mm t``^{-1}``]
+taken from the ``U`` store, available water in the ``S`` store (`satwaterdepth` [mm]) and
+the deficit of the ``U`` store (`ustorecapacity` [mm]), as shown by the following code
+block:
 
-```math
-CSF=CS/(CS+z_{i}-RT)
+```julia
+    maxcapflux = max(0.0, min(ksat, actevapustore, ustorecapacity, satwaterdepth))
+```
+
+Then the potential rise `maxcapflux` is scaled using the distance between the roots and the
+water table as follows in the code block below (`i` refers to the Vector position):
+
+```julia
+    if sbm.zi[i] > rootingdepth
+        capfluxscale =
+            sbm.capscale[i] / (sbm.capscale[i] + sbm.zi[i] - rootingdepth) *
+            (sbm.Δt / tosecond(basetimestep))
+    else
+        capfluxscale = 0.0
+    end
+    capflux = maxcapflux * capfluxscale
+```
+
+where `capfluxscale` [-] is the scaling factor to multiply the potential rise `maxcapflux`
+with, `sbm.capscale` [mm] is a model parameter (default = 100) and `rootingdepth` [mm] the
+rooting depth. If the roots reach the water table (`rootingdepth` ``\ge`` `sbm.zi`),
+`capfluxscale` is set to zero and thus setting the capillary rise `capflux` to zero.
+
+Finally, the capillary rise `capflux` is limited by the unsaturated store deficit (one or
+multiple layers), calculated as follows in the code block below (`i` refers to the Vector
+position, and `k` refers to the layer position):
+
+```julia
+    usl[k] * (sbm.θₛ[i] - sbm.θᵣ[i]) - usld[k]
 ```
-in which ``CSF`` is the scaling factor to multiply the potential rise with, ``CS`` is a
-model parameter (default = 100) and ``RT`` the rooting depth. If the roots reach the water
-table (``RT > z_{i}``) ``CS`` is set to zero thus setting the capillary rise to zero.
+
+where `usl` [mm] is the unsaturated layer thickness, `usld` is the `ustorelayerdepth` \[mm\]
+(amount of water in the unsaturated layer), and ``\theta_{s}`` and ``\theta_{r}`` as
+previously defined.
+
+The calculation of the actual capillary rise `actcapflux` is as follows in the code block
+below (`i` refers to the Vector position, and `k` refers to the layer position):
+
+```julia
+    actcapflux = 0.0
+    netcapflux = capflux
+    for k = n_usl:-1:1
+        toadd =
+            min(netcapflux, max(usl[k] * (sbm.θₛ[i] - sbm.θᵣ[i]) - usld[k], 0.0))
+        usld = setindex(usld, usld[k] + toadd, k)
+        netcapflux = netcapflux - toadd
+        actcapflux = actcapflux + toadd
+    end
+```
+
+In case of multiple unsaturated layers (`n_usl` ``>`` 1), the calculation of the actual
+capillary rise starts at the lowest unsaturated layer while keeping track of the remaining
+capillary rise `netcapflux` [mm t``^{-1}``].
 
 ## Leakage
-If the `maxleakage` parameter is set > 0, water is lost from the saturated zone and runs out
-of the model.
+If the `maxleakage` [mm/day] input model parameter is set > 0, water is lost from the
```

Comment:
[mm t``^{-1}``] instead of mm/day for leakage?

Reply:
Input model parameter that represents a flux has always time unit of day (base timestep of Wflow).

Reply:
ah good point thanks!